evalml/tests/automl_tests/parallel_tests/test_automl_dask.py

```diff
@@ -309,3 +309,35 @@ def test_automl_immediate_quit(
         assert DaskPipelineWithFitError.custom_name not in set(
             automl.full_rankings["pipeline_name"]
         )
+
+
+@pytest.mark.parametrize(
+    "engine_str", ["cf_threaded", "cf_process", "dask_threaded", "dask_process"]
+)
+def test_automl_convenience(engine_str, X_y_binary_cls):
```

Comment:
nit-pick: I think adding something about the engine str to the test name makes it clearer what the test is actually testing.